test/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM cypress/browsers:node16.5.0-chrome94-ff93
```

Comment:
For added security (and self documentation) we usually prefer to extend one of our base images for things like this. How much more work would it be to start from the "CLARIN Alpine base image", install the necessary cypress packages, and proceed with the remaining Dockerfile instructions?

Reply:
I see your point, I'll do that

Reply:
A good candidate for base will be: https://gitlab.com/CLARIN-ERIC/docker-alpine-base

Reply:
Hmmm, it seems cypress does not work on alpine yet (https://github.com/cypress-io/cypress/issues/419). Do you have a preference for another base?

Reply:
open since Fev 2017! I am not sure, but if we can't use cypress for more browsers than Chrome, and we cannot run it on Alpine due to an issue reported in 2017 which is still open, I start having doubts this is a tool that fits our infrastructure.
Maybe leave it as is for now.

Reply:
It's also possible to use it with firefox; otherwise, it's as you said.

Reply:
Lets leave it as it is for now.
It seems like cypress is lacking a slim image in general. browsers:node16.5.0-chrome94-ff93 is [~750Mb](https://hub.docker.com/layers/cypress/browsers/node16.5.0-chrome94-ff93/images/sha256-a5af59c9526fb9a885fb6cfd58904d50940843b7a5ed5eec2affbf6d736deb1f?context=explore)
So there is not much point in going after an Alpine-like image

Reply:
There is also this as alternative to the whole docker setup for UI testing: https://github.com/marketplace/actions/cypress-io 
Did you do some evaluation on it?

Reply:
Yes, it's pretty much the same thing (it's also based on a docker image). We cannot use it directly because in our case the backend is not started by npm but through java, so we'd need additional packages installed and so on.